binding/SkiaSharp.Resources/ResourceProvider.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+
+namespace SkiaSharp.Resources
+{
+	public abstract unsafe class ResourceProvider : SKObject, ISKReferenceCounted, ISKSkipObjectRegistration
```

Comment:
Note to future self. Don't forget to correctly apply the `ISKReferenceCounted` interface and NOT explicitly delete the reference counted types. This causes a hard-to-debug crash because the type is deleted while in use. Ot sometimes crashes in a debug build with a better log message, but not often.